scanner/src/main/kotlin/LocalScanner.kt

```diff
@@ -270,17 +270,15 @@ abstract class LocalScanner(name: String, config: ScannerConfiguration) : Scanne
                 }
 
                 if (config.createMissingArchives) {
-                    val packagePath = pkg.id.toPath()
```

Comment:
Just a small nit: This variable existed so that the `pkg.id.toPath()` does not have to be called for each `storedResult` that has not been archived yet. Probably just a small speedup.

Reply:
Thanks @KorSin! I consciously decided against this optimization to enable removing some code duplication. This is done in a following commit.